lib/devise_security_extension/models/mongoid/old_password.rb

```diff
@@ -0,0 +1,12 @@
+class OldPassword
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :encrypted_password, :type => String
+  field :password_salt, :type => String
+  field :password_archivable_id, :type => Integer
+  field :password_archivable_type, :type => String
+
+  belongs_to :password_archivable, :polymorphic => true
+  attr_accessible :encrypted_password
+end
```

Comment:
Two comments on this model (and I'm not a Ruby on Rails expert, so take this with a grain of salt but):

1. If you have the `field` for "encrypted_password, why would you need the `attr_accessible`? Isn't that redundant to some degree?
2. Similarly, wouldn't defining `belongs_to :password_archivable, :polymorphic` automatically created the `password_archivable_id` and `password_archivable_type` fields?